README.md

```diff
@@ -23,9 +23,19 @@
 
 ## Being a better company to work at
 
+- [ ] Setup the ML cluster environment
 - [ ] Regression Test, Benchmarks and Deploy platform (CD?)
+    - For which projects? Are any of ML included? - Vadim
 - [ ] Improve Logs, CLI, and other guides.
+    - Logs for Go or for Python too? CLI guide - do we have one?
 - [ ] Make it easy to update and maintain PGA.
+- [ ] Conduct a happiness survey and react to the unveiled issues
+- [ ] Improve the formal process of speaking at a conference
```

Comment:
This is not a OKR

Reply:
What is it instead?

Reply:
This is some that you should speak with DevRel team.